doc/user/COMMANDS.md

```diff
@@ -41,7 +41,7 @@ Supports `front`, `top`, `right`, `isometric` arguments. eg: `set_camera top`.
 
 `toggle_volume_rendering`: A specific command to toggle `model.volume.enable` and print coloring information. No argument.
 
-`stop_interactor`: A specific command to stop the interactor hence quitting the application. No argument.
+`exit`: A specific command to stop the interactor hence quitting the application. No argument.
```

Comment:
@mwestphal I decided to change the command name because it's not associated to any binding now. Since it's only accessible from the console, typing `exit` is more convenient.

Reply:
Alright